src/Hubbup.Web/Pages/Index.cshtml

```diff
@@ -0,0 +1,308 @@
+@page "{groupName}"
+@model IndexModel
+@{
+    ViewData["PageTitlePrefix"] = Model.GroupName;
+    ViewData["CurrentGroup"] = Model.GroupName;
+}
+
+<div class="row issue-list-row">
+    <div class="col-md-2 col-md-push-10">
+        <a asp-controller="IssueList" asp-action="Index" asp-route-repoSet="@Model.GroupName">Switch to Legacy View</a>
+        <h3>Rate Limit</h3>
+        <div id="rate-limit-display"></div>
+    </div>
+
+    <div class="col-md-10 col-md-pull-2">
+        <div class="tab-content">
+            <div class="tab-pane active" id="group-by-assignee">
+                @foreach (var person in Model.People)
+                {
+                    <div class="panel panel-default i-assignee-panel" data-assignee="@person" data-id="@TagBuilder.CreateSanitizedId(person, "-")">
+                    </div>
+                }
+            </div>
+        </div>
+    </div>
+</div>
+
+@section scripts {
+    <script id="rate-limit-template" type="text/x-handlebars-template">
+        {{#if graphql}}
+            <div class="alert {{graphql.alertClass}} small">GraphQL Rate Limit Remaining: {{graphql.remaining}}</div>
+        {{/if}}
+        {{#if rest}}
+            <div class="alert {{rest.alertClass}} small">REST Rate Limit Remaining: {{rest.remaining}}</div>
+        {{/if}}
+        {{#if globalCost}}
+            <div class="alert alert-info small">GraphQL Rate Limit cost since last page load: {{globalCost}}</div>
+        {{/if}}
+    </script>
+
+    <script id="assignee-panel-template" type="text/x-handlebars-template">
+        <div class="panel-heading">
+            {{#if rateLimit}}
+                <div class="pull-right small">
```

Comment:
Moving it. Also going to remove the rate limit badges from the user frames in non-Development environments. They're really useful when developing to see the rate limit impact of each load, but just noise in "Production"